examples/demo/config/development.yaml

```diff
@@ -156,3 +156,35 @@ auth:
     expiration: 604800 # 7 days
 # </snip>
 
+# Scheduler Jobs Configuration
+# <snip id="configuration-scheduler">
+scheduler:
+  # Location of shipping the command stdout and stderr.
+  output: stdout
+  # A list of jobs to be scheduled.
+  jobs:
+      # The name of the job.
+    - name: "Run command"
```

Comment:
use a hashmap instead of vec, id instead of name

```
jobs:
   send_emails:
         ...
```

Reply:
agree. 
changed: https://github.com/loco-rs/loco/pull/735/commits/19d2a0f8b071e2cc56df893dc3432af60fc91026